### PR TITLE
Improved grammar and readability.

### PR DESCRIPTION
**Pull request by yarwest**
Tried to make every sentence as clear as possible. Referencing issue #42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1 align="center">Frameworks code comparison</h1>
+<h1 align="center">Web framework code comparison</h1>
```
Comment (feimosi) [inline README.md]: I'd leave the title as it is.

File: README.md [hunk 2]
```diff
@@ -8,9 +8,9 @@
 
 </div>
 
-Comparison of different approaches in writing web applications. Based on React, Angular, AngularJS and Vue.js. Useful when migrating between frameworks or switching projects often.
+Comparison of different approaches in writing web applications. Based on React, Angular, AngularJS and Vue.js. This comparison is useful when migrating between frameworks or switching projects often.
```
Comment (feimosi) [inline README.md]: I think `It's especially useful` fits better here.

File: README.md [hunk 3]
```diff
@@ -8,9 +8,9 @@
...
 
-All examples follow the current best practices and conventions inside the given framework community. Angular code is written in TypeScript.
+All examples follow the current best practices and conventions that are used inside the community of the given framework. Angular code is written in TypeScript.
```
Comment (feimosi) [inline README.md]: `of a given framework`

File: README.md (hunk 1 above)
Comment (feimosi) [reply README.md]: Possible could also be: `Frontend frameworks code comparison`

**Review by feimosi — CHANGES_REQUESTED** [review]
Great work, thanks a lot!
Take a look at my comments.

**Comment by yarwest** [comment]
I have made the requested changes :)

**Comment by feimosi** [comment]
Looks good :)

**Pull request merged**